interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
hi @robertdown ,
I am unable to use bootstrap's nice alert/alert-* classes since we are overriding alert with the bright red alert box. Can you think of a way I can avoid this issue and be able to use bootstrap's alert classes?
thanks, -brady